luigi/tools/deps-tree.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from luigi.task import flatten
+from luigi.cmdline_parser import CmdlineParser
+import sys
```

Comment:
For some inspiration, see the docs for [execution summary example](https://github.com/spotify/luigi/blob/5f570ad8484f55382267ff2727d2df6baa4c6f97/examples/execution_summary_example.py#L16-L46)